### PR TITLE
feature: make bm25 sparse vector

### PR DESCRIPTION
**Pull request by cdxker**

### CODE REVIEW

**Review by densumesh — APPROVED** [review]

**Review by skeptrunedev — CHANGES_REQUESTED** [review]
BM25 should be active by default for a dataset. We should use default values for the constants instead. 

If someone in an org for a dataset does nothing then they should still get BM25.

**Comment by cdxker** [comment]
Just updated it to on the search UI

**Review by skeptrunedev — CHANGES_REQUESTED** [review]
Add into `.env.search` for the `VITE_BM25_ACTIVE` env

File: .env.dashboard [hunk 1]
```diff
@@ -3,3 +3,4 @@ VITE_SEARCH_UI_URL="http://localhost:5174"
 VITE_CHAT_UI_URL="http://localhost:5173"
 VITE_ANALYTICS_UI_URL="http://localhost:5176"
 VITE_SENTRY_DASHBOARD_DSN=""
+VITE_BM25_ACTIVE=false
```
Comment (skeptrunedev) [inline .env.dashboard:6]:
```suggestion
VITE_BM25_ACTIVE=true
```

File: .env.search [hunk 2]
```diff
@@ -2,3 +2,4 @@
 VITE_API_HOST=http://localhost:8090/api
 VITE_SENTRY_SEARCH_DSN=""
 VITE_DASHBOARD_URL=http://localhost:5173
+VITE_BM25_ACTIVE=false
```
Comment (skeptrunedev) [inline .env.search:5]:
```suggestion
VITE_BM25_ACTIVE=true
```

**Review by skeptrunedev — CHANGES_REQUESTED** [review]

**Review by skeptrunedev — CHANGES_REQUESTED** [review]
Need to have BM25 in update

**Review by skeptrunedev — APPROVED** [review]

**Pull request merged**